<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageEdit</class>
 <widget class="QDialog" name="ImageEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图像编辑</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resourse.qrc">
    <normaloff>:/ico/icon/logo.png</normaloff>:/ico/icon/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <widget class="ImageView" name="graphicsView">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="topFrame">
     <property name="styleSheet">
      <string notr="true">QFrame
{
	margin:0px;
	padding:0px;
	background-color: rgb(17,28,27);
}

QToolButton
{
	background-color: rgb(16,42,57);
	color:rgb(255,255,255);
	border:none;
}

QToolButton:checked
{
	background-color: rgb(0,122,204);
	color:rgb(255,255,255);
	border:none;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="tbOpen">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>打开</string>
        </property>
        <property name="icon">
         <iconset resource="resourse.qrc">
          <normaloff>:/ico/icon/open.png</normaloff>:/ico/icon/open.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbSave">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
        <property name="icon">
         <iconset resource="resourse.qrc">
          <normaloff>:/ico/icon/SaveAs.png</normaloff>:/ico/icon/SaveAs.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbUndo">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>撤销</string>
        </property>
        <property name="icon">
         <iconset resource="resourse.qrc">
          <normaloff>:/ico/icon/undo.png</normaloff>:/ico/icon/undo.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbRedo">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>重做</string>
        </property>
        <property name="icon">
         <iconset resource="resourse.qrc">
          <normaloff>:/ico/icon/redo.png</normaloff>:/ico/icon/redo.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbZoomIn">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>缩小</string>
        </property>
        <property name="icon">
         <iconset resource="resourse.qrc">
          <normaloff>:/ico/icon/zoomin.png</normaloff>:/ico/icon/zoomin.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbZoomOut">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>放大</string>
        </property>
        <property name="icon">
         <iconset resource="resourse.qrc">
          <normaloff>:/ico/icon/zoom.png</normaloff>:/ico/icon/zoom.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbZoom">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>100%</string>
        </property>
        <property name="icon">
         <iconset resource="resourse.qrc">
          <normaloff>:/ico/icon/fitscreen.png</normaloff>:/ico/icon/fitscreen.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbResize">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>尺寸</string>
        </property>
        <property name="icon">
         <iconset resource="resourse.qrc">
          <normaloff>:/ico/icon/resize.png</normaloff>:/ico/icon/resize.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbCrop">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>裁剪</string>
        </property>
        <property name="icon">
         <iconset resource="resourse.qrc">
          <normaloff>:/ico/icon/crop.png</normaloff>:/ico/icon/crop.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbCopy">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>剪切板</string>
        </property>
        <property name="icon">
         <iconset resource="resourse.qrc">
          <normaloff>:/ico/icon/copy.png</normaloff>:/ico/icon/copy.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="leftFrame">
     <property name="styleSheet">
      <string notr="true">QFrame
{
	margin:0px;
	padding:0px;
	background-color: rgb(17,28,27);
}

QToolButton
{
	background-color: rgb(16,42,57);
	color:rgb(255,255,255);
	border:1px solid rgb(16,42,57);
}

QToolButton:checked
{
	background-color: rgb(16,42,57);
	color:rgb(255,255,255);
	border:1 solid rgb(0,122,204);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="tbMove">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resourse.qrc">
          <normaloff>:/ico/icon/move.png</normaloff>:/ico/icon/move.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbText">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resourse.qrc">
          <normaloff>:/ico/icon/text.png</normaloff>:/ico/icon/text.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbArrow">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resourse.qrc">
          <normaloff>:/ico/icon/arrow.png</normaloff>:/ico/icon/arrow.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbPen">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resourse.qrc">
          <normaloff>:/ico/icon/pen.png</normaloff>:/ico/icon/pen.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbRect">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resourse.qrc">
          <normaloff>:/ico/icon/rect.png</normaloff>:/ico/icon/rect.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="tbSelColor">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 0, 0);
border:1px solid rgb(16,42,57);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header location="global">imageview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
